src/ground_truth_nsrts.py

```diff
@@ -282,6 +283,34 @@ def place_sampler(state: State, rng: np.random.Generator,
                       set(), option, option_vars, place_sampler)
     nsrts.add(place_nsrt)
 
+    # Place (not on any target)
+    if CFG.env == "cover_nonrefinable":
+        parameters = [block]
+        preconditions = {
+            LiftedAtom(IsBlock, [block]),
+            LiftedAtom(Holding, [block])
+        }
+        add_effects = {
+            LiftedAtom(HandEmpty, []),
+        }
+        delete_effects = {LiftedAtom(Holding, [block])}
+        option = PickPlace
+        option_vars = []
+
+        def place_on_table_sampler(state: State, rng: np.random.Generator,
+                                   objs: Sequence[Object]) -> Array:
+            del state  # unused
+            assert len(objs) == 1
+            lb = 0.0
+            ub = 1.0
+            return np.array(rng.uniform(lb, ub, size=(1, )), dtype=np.float32)
```

Comment:
so this isn't a perfect sampler, then? will that have any bad implications? i guess not as long as we're planning to keep max_samples_per_step = 10. but it would be a problem if we change that to 1 to allow ourselves a higher skeleton budget. don't think we need the higher skeleton budget for cover though.